src/init/starship.zsh

```diff
@@ -66,15 +66,15 @@ starship_zle-keymap-select() {
 }
 
 ## Check for existing keymap-select widget.
-local existing_keymap_select_fn=$widgets[zle-keymap-select];
+__starship_preserved_zle_keymap_select=$widgets[zle-keymap-select];
 # zle-keymap-select is a special widget so it'll be "user:fnName" or nothing. Let's get fnName only.
-existing_keymap_select_fn=${existing_keymap_select_fn//user:};
-if [[ -z $existing_keymap_select_fn ]]; then
+__starship_preserved_zle_keymap_select=${__starship_preserved_zle_keymap_select//user:};
```

Comment:
We can avoid reassignment by converting it to:
```sh
__starship_preserved_zle_keymap_select=${widgets[zle-keymap-select]#user:}
```

Reply:
Updated.